docs/ensembl-help/using-ensembl/ensembl-apps/entity-viewer/gene-function.md

```diff
@@ -12,14 +12,38 @@ tags:
 status: draft
 ---
 
-# What’s in the gene function?
-
-Gene function view presents protein information for all coding transcripts of the gene of interest. It is therefore only available for protein coding genes.
- 
-By default, the protein list is sorted by the amino acid length and specifies the protein name and the Ensembl protein stable ID with its corresponding transcript stable ID. You can show/hide extra information by clicking on the protein of interest. The top line is expanded by default.
- 
-Expanded view displays a graphical representation of the focus protein (shown in orange) and position of its domains (dark grey boxes) relative to the longest translation (in light grey). You can find the protein domain source on the left and its description on the right.
- 
-External links to matching protein entries available for the expanded protein are listed below the diagram. Those include records from the EBI protein resources, such as InterPro, UniProt and PDBe-KB. PDBe-KB entry types and counts are illustrated by black icons.
- 
-You can also export the amino acid and coding sequence (CDS) for the expanded translation by ticking the relevant box(es) and clicking on the green Download button.
+# What’s in the gene function panel?
+
+The Gene function panel presents protein information for all coding transcripts of the gene of interest. It is therefore only available for protein coding genes.
+
+Proteins are listed in order of amino acid length, by default.  
+
+Each protein is represented by its name, Ensembl protein stable ID with its corresponding transcript stable ID in addition to their amino acid length.
+
+## Detailed protein information
+
+The detailed information for the Focus protein is shown by default.
+
+You can show/hide extra information by selecting the Ensembl transcript stable ID for a protein of interest. 
+
+Focus protein
+
+The Focus protein is produced from the MANE Select or Canonical transcript.
+
+The expanded view is a graphical representation of the Focus protein.
+
+The full length of the protein is shown in orange. 
+
+The light grey section represents the longest translation of the protein. 
```

Comment:
'represents the length of the translation of the longest coding transcript of the gene'

Reply:
done